lib/api.d.ts

```diff
@@ -0,0 +1,40 @@
+export declare function log(message: string): void;
+export declare function signerAccountId(): string;
+export declare function signerAccountPk(): string;
+export declare function predecessorAccountId(): string;
+export declare function blockIndex(): BigInt;
+export declare function blockHeight(): BigInt;
+export declare function blockTimestamp(): BigInt;
+export declare function epochHeight(): BigInt;
+export declare function attachedDeposit(): BigInt;
+export declare function prepaidGas(): BigInt;
+export declare function usedGas(): BigInt;
+export declare function randomSeed(): string;
+export declare function sha256(value: string): string;
```

Comment:
I suggest to typedef string to bytes, then set those API taking bytes with that interface, such as sha256, storageWrite, logUtf8 etc. Some APIs are takes/returns string, such as jsvmMethodName. Could you use reference this for which ones are bytes: https://github.com/near/near-sdk-js#near-sdk-js-api-reference

Reply:
I suppose it's fine to alias this for now. Just feels weird that there are these semantics that is specific to this project and not expressed in some way through the type system to avoid misuse